bot/utils/extensions.py

```diff
@@ -0,0 +1,34 @@
+import importlib
+import inspect
+import pkgutil
+from typing import Iterator, NoReturn
+
+from bot import exts
+
+
+def unqualify(name: str) -> str:
+    """Return an unqualified name given a qualified module/package `name`."""
+    return name.rsplit(".", maxsplit=1)[-1]
+
+
+def walk_extensions() -> Iterator[str]:
+    """Yield extension names from the bot.exts subpackage."""
+
+    def on_error(name: str) -> NoReturn:
+        raise ImportError(name=name)  # pragma: no cover
+
+    for module in pkgutil.walk_packages(exts.__path__, f"{exts.__name__}.", onerror=on_error):
+        if unqualify(module.name).startswith("_"):
```

Comment:
In the `snakes` cog folder, you'll need to prefix all the file names with `_`, so they each won't be counted as a cog. Instead the whole `snakes` folder will be counted as a cog.

We want this: 
![image](https://user-images.githubusercontent.com/52584399/93817628-3a48e000-fc27-11ea-8c08-b3813b8bdd2a.png)

Instead of this:
![image](https://user-images.githubusercontent.com/52584399/93817698-4f257380-fc27-11ea-9e40-2d4cf39d1666.png)